<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>711</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load_csv">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>580</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #cf617a; /* Розовый цвет фона */
    color: white; /* Белый цвет текста */
    border: none; /* Убрать рамку */
    border-radius: 20px; /* Закругление углов для овала */
     padding: 10px 20px; /* Внутренние отступы */
    font-size: 16px; /* Размер шрифта */
    /* Задаем овальную форму */
}

QPushButton:hover {
    background-color: #c36077; /* Цвет фона при наведении */
}

QPushButton:pressed {
    background-color: #703845; /* Цвет фона при нажатии */
}</string>
   </property>
   <property name="text">
    <string>Выгрузка из CSV</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_year">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>641</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/обои.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>tableWidget</zorder>
  <zorder>pushButton_load_csv</zorder>
  <zorder>comboBox_year</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
